openpype/pipeline/create/context.py

```diff
@@ -1458,6 +1458,65 @@ def create(self, identifier, *args, **kwargs):
                     identifier, label, exc_info, add_traceback
                 )
             ])
+        return result
+
+    def create(
+        self,
+        creator_identifier,
+        variant=None,
+        asset_doc=None,
+        task_name=None,
+        pre_create_data=None
+    ):
+        """Trigger create of plugins with standartized
+
+        Args:
+            creator_identifier (str):
+            variant (str): Variant used for subset name.
+            asset_doc (Dict[str, Any]):
+            task_name (str): Name of task to which is context related.
+            pre_create_data (Dict[str, Any]): Pre-create attribute values.
+
+        Returns:
+            Any: Output of triggered creator's 'create' method.
+
+        Raises:
+            CreatorsCreateFailed: When creation fails.
+        """
+
+        if pre_create_data is None:
+            pre_create_data = {}
+
+        project_name = self.project_name
+        if asset_doc is None:
+            asset_name = self.get_current_asset_name()
+            asset_doc = get_asset_by_name(project_name, asset_name)
+            task_name = self.get_current_task_name()
```

Comment:
I feel like we might only want to do this if `task_name` is None?
Additionally however, it is technically possible to supply no task name to a creator too?

Reply:
If it's a special case, let's clarify in the code/docstring yes!

Reply:
Or, make them required in this method and add new method where is explicitly used current context?